lightning.install

```diff
@@ -9,6 +9,34 @@ use Drupal\Core\Entity\Entity\EntityViewDisplay;
 use Drupal\field\Entity\FieldConfig;
 use Drupal\lightning_core\ConfigHelper as Config;
 
+/**
+ * Alerts users of Libraries API dependencies.
+ *
+ * A future version of Lightning will remove Libraries API, so
+ * users must add that dependency to composer.json manually.
+ */
+function lightning_requirements() {
+    $requirements = [];
+    $libraries_dependencies = [];
+    $enabled_modules = \Drupal::moduleHandler()->getModuleList();
+    
+    foreach ($enabled_modules as $module => $data) {
```

Comment:
This can be a bit simpler; the module extension list has a `getAllInstalledInfo()` method which will return info for all installed modules.